ChangeLog

```diff
@@ -10,6 +10,8 @@ Bugfix
    * Add ecc extensions only if an ecc based ciphersuite is used.
      This improves compliance to RFC 4492, and as a result, solves
      interoperability issues with BouncyCastle. Raised by milenamil in #1157.
+   * Fix potential segmentation fault in mbedtls_ssl_get_max_frag_len()
```

Comment:
Why would there be a segmentation fault? Please be more specific. Null pointer dereference?

Reply:
@gilles-peskine-arm Ok, it would be more precise to say that it's a potential use-after-free.

Reply:
Now that Gilles mentions it, I'm thinking perhaps we should always avoid the wording "segmentation fault" regardless of the cause, as it's a platform-specific behaviour.